src/DB/exerciseData.json

```diff
@@ -36,7 +38,10 @@
     ],
     "videoLink": "https://www.youtube.com/watch?v=1fbU_MkV7NE",
     "gh-name": "jayasree-g",
-    "image": "https://www.fitstream.com/images/bodyweight-training/bodyweight-exercises/sit-up-large.png"
+    "image": "https://www.fitstream.com/images/bodyweight-training/bodyweight-exercises/sit-up-large.png",
+    "tags": ["Abdominals"]
```

Comment:
I think it’s best to move all the tags to a separate file. We can import the tags here and use. This will ensure non duplication of tags. By this way we reuse, the string also.